### DISCUSSION TITLE
Taiko maintainance

### DISCUSSION THREAD

**Discussion by ravikiranhebbar88** [getgauge/taiko#2563] [General]
As we have created the taiko framework and all the scenarios are reliable most of the time. I would like to ask whether there is any plan in stopping this taiko maintenance in 1 or 2yrs. As we have more than 2000+ scenarios, we are looking to migrate the scenarios to this newly created taiko framework.

**Comment by zabil** [comment]
Unfortunately, there is no clear answer to this. Taiko maintenance is a volunteer effort.  
Please read https://github.com/getgauge/gauge/issues/1732

**Reply by ravikiranhebbar88** [reply]
> Unfortunately, there is no clear answer to this. Taiko maintenance is a volunteer effort. Please read [getgauge/gauge#1732](https://github.com/getgauge/gauge/issues/1732)

Thankyou zabil. Taiko is one of the best libraries compared to other automation tools and is one of my favorite :)
